canonical-websites/www.ubuntu.com
# Fix urgent typos on the Short Form Services Agreement

## Done

Fix urgent typos on the Short Form Services Agreement 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/short-terms](http://0.0.0.0:8001/legal/short-terms)
- Make sure the changes outlined in the doc have been made

## Issue / Card

Doc: https://docs.google.com/document/d/1W0WJrXaIvC54MDPgS62xfgPhqJaSE7RcZ2mjIqz1Y2o/edit#

Fixes #2409
